lib/pacemaker/pcmk_sched_probes.c

```diff
@@ -446,6 +446,63 @@ add_probe_orderings_for_stops(pe_working_set_t *data_set)
     }
 }
 
+/*!
+ * \internal
+ * \brief Add necessary orderings between probe and starts of clone instances
+ *
+ * , in additon to the ordering with the parent resource added upon creating
+ * the probe.
+ *
+ * \param[in] probe     Probe as 'first' action in an ordering
+ * \param[in] after     'then' action wrapper in the ordering
```

Comment:
We're now marking pointer arguments as [in,out] if anything reachable via the pointer (directly or indirectly) can change, and order_actions() will modify both of these